src/NHN.DtoContracts/NHN.DtoContracts/Krk/Data/KommuneoverlegeInfo.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.ComponentModel.DataAnnotations;
+using NHN.DtoContracts.Common.en;
+
+namespace NHN.DtoContracts.Krk.Data
+{
+    /// <summary>
+    /// Helseregister objekt for å legge til/hente ut opplysninger om en helseregisteroppføring i OFR
+    /// </summary>
+    [DataContract(Namespace = Namespaces.KrkV1)]
+    public class KommuneoverlegeInfo
+    {
+
+        /// <summary>
+        /// ReshId til kommunen som eier kommuneoverlege-tjenesten (KA03)
+        /// </summary>
+        [DataMember]
+        public int OwnerReshId { get; set; }
+
+        /// <summary>
+        /// Bydel/kommunen kommuneoverlegen tilhører
+        /// </summary>
```

Comment:
Jeg skjønner at bydel impliserer en gitt kommune, men er det ikke likevel lettere å forholde seg til om bydel og kommune er to distinkte properties i stedet for å blande to ulike kodetyper i én property?